frontend/src/app/settings/modelsources/git-settings/git-settings.component.html

```diff
@@ -4,34 +4,36 @@
  -->
 
 <div class="wrapper">
-  <div class="title" *ngIf="availableGitSettings.length">
-    Existing instances
-  </div>
-  <div class="instances" *ngIf="availableGitSettings.length">
-    <mat-card *ngFor="let instance of availableGitSettings">
-      <div class="mat-title">
-        <div class="instance-head">
-          {{ instance.name }}
-          <span class="mat-subtitle">{{ instance.type }}</span>
+  <div *ngIf="(this.gitSettingsService.gitSettings | async)?.length !== 0">
```

Comment:
```suggestion
  <div *ngIf="(this.gitSettingsService.gitSettings | async)?.length">
```

This should do the same.